wordpress-mobile/WordPress-iOS
# ULS Password view: update Auth to add VoiceOver support

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/352
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/378


**Prerequisites:**
- Google account with a WP password set.
- Enable the `.unifiedGoogle` feature flag.

**To test:**
- Log In > Continue with Google.
- Enable VoiceOver.
- Complete Google auth.
- On the Password view:
  - VO should read the password field: `Password. Secure textfield. Is editing. Character mode. Insertion point at start.`
  - The 1Password button on the email field should now read: `One Password. Button. Opens One Password manager.`. (I have no idea what it said before. It was unintelligible.)
  - The other components should read their text.
  - Enter an invalid password. VoiceOver should focus on the error message and read it when the UI changes.

PR submission checklist:
